### PR TITLE
Revamp/dgraph overview 351

### PR DESCRIPTION
**Pull request by damonfeldman**

### CODE REVIEW

File: content/dgraph-overview.md [hunk 1]
```diff
@@ -1,150 +1,149 @@
...
+
+- **GraphQL Support**: &emsp; *GraphQL is built in to make data access simple and standards-compliant. Unlike most GraphQL solutions, no resolvers are needed - Dgraph resolves queries automatically through graph navigation.*
+
+- **True Free Open Source Software (FOSS)**: &emsp; *Dgraph is free to use, and [available on github](https://github.com/dgraph-io/dgraph).*
+
+- **Fully Transactional and ACID Compliant**: &emsp; *Dgraph satisfies demanding OLTP workloads that require frequent inserts and updates.*
+
+- **Text Search**: &emsp; *Full-text searching is included.*
+## Who is using Dgraph & How
+Dgraph is a graph database with some special differentiators. Any graph database will be better when relationships and connections really matter, such as when querying across links in a social network, or detecting networks of interconnected fraudsters in a fraud application - these are natural graph use cases.
```
[the remark this answers is not in the document]
Comment (Rajakavitha1) [reply content/dgraph-overview.md:25]: Thank you for the rewrite of the overview @damonfeldman . Just minor edits to avoid lengthy sentence.

File: content/design-concepts/concepts.md [hunk 2]
```diff
@@ -6,37 +6,7 @@ weight = 3
     parent = "design-concepts"
 +++
 
-## Edges
```
Comment (MichelDiz) [inline content/design-concepts/concepts.md:9]: Why remove this? This explains how an edge/triple works and demonstrates that it is a directed edge.

File: content/get-started/index.md [hunk 3]
```diff
@@ -1,166 +1,107 @@
...
 +++
 
+This is a quickstart guide to run Dgraph. For an interactive walkthrough, take the [tour](https://dgraph.io/tour/).
+
 {{% notice "note" %}}
-This is a quick start guide.
-You can find the getting started tutorial series [here]({{< relref "tutorials/index.md" >}}).
+[DQL](https://dgraph.io/docs/main/query-language/#graphql) is a powerful query language of Dgraph,
+which is a variation of [GraphQL](https://graphql.org/), a query language created by Facebook.
+For GraphQL quickstart information see, [dgraph.io/graphql](https://dgraph.io/graphql). This guide is not recommended for production environment.
```
Comment (MichelDiz) [inline content/get-started/index.md:16]: >This guide is not recommended for production environment. 

What it means? Any guide is a potential content to learn and use in prod. I don't see why mention this. Nobody is going to use tutorials and guides in production.

File: content/tasks/start-dgraph-server.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+#### To run Dgraph using the standalone Docker image
+
+1. Download docker: https://www.docker.com/
+2. Create a folder to store Dgraph data outside of the container, as follows: `mkdir -p ~/dgraph`
+3. Get the Docker standalone image, as follows: `docker pull dgraph/standalone`
+4. Run the Dgraph Docker standalone image, as follows:
+
+```sh
+  docker run -it -p 5080:5080 -p 6080:6080 -p 8080:8080 -p 9080:9080 -v ~/dgraph:/dgraph --name dgraph dgraph/standalone:{{< version >}}
```
Comment (MichelDiz) [inline content/tasks/start-dgraph-server.md:27]: In this case, we shouldn't expose zero. Let's remove `docker run -it`.

File: content/tasks/start-dgraph-server.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+```  
+
+{{% notice "tip" %}}
+To run the Docker standalone image for another version of Dgraph, change `v21.03.0`
+in the command shown above to the version number for a previous release, such as `v20.11.0`.
+{{% /notice %}}
```
Comment (MichelDiz) [inline content/tasks/start-dgraph-server.md:33]: I believe this note is out of scope. Those who use Docker should know that it is possible to use an old version by changing the tag. But we shouldn't recommend this practice for novices.

File: content/get-started/index.md [hunk 6]
```diff
@@ -1,166 +1,107 @@
...
   weight = 2
 +++
 
+This is a quickstart guide to run Dgraph. For an interactive walkthrough, take the [tour](https://dgraph.io/tour/).
+
 {{% notice "note" %}}
-This is a quick start guide.
-You can find the getting started tutorial series [here]({{< relref "tutorials/index.md" >}}).
+[DQL](https://dgraph.io/docs/main/query-language/#graphql) is a powerful query language of Dgraph,
+which is a variation of [GraphQL](https://graphql.org/), a query language created by Facebook.
```
Comment (MichelDiz) [inline content/get-started/index.md:15]: I don't know if the word "variation" fits here. DQL is an inspiration from GraphQL.

File: content/tasks/start-dgraph-server.md (hunk 4 above)
Comment (MichelDiz) [reply content/tasks/start-dgraph-server.md:27]: remove `-p 5080:5080 -p 6080:6080` ****

**Comment by rderbier** [comment]
@damonfeldman please see what should be included in this ovreview from Akon post : https://www.linkedin.com/posts/akon-dey_earlystage-graphql-knowledgegraph-activity-7014349338667536385-fFOK?utm_source=share&utm_medium=member_desktop

File: content/design-concepts/concepts.md (hunk 2 above)
Comment (damonfeldman) [reply content/design-concepts/concepts.md:9]: We do need to define them, but are moving to "relationships" which will be added to the glossary. This particular content is about N-quads, which is really just an input format we support (along with JSON) and not fundamental, along with internal golang data structures which if we keep should be in some developer contributing doc, vs the actual DB docs.

File: content/tasks/start-dgraph-server.md (hunk 4 above)
Comment (damonfeldman) [reply content/tasks/start-dgraph-server.md:27]: Great. We moved this to "how to" and removed the 5080 and 6080 ports.

**Comment by rderbier** [comment]
was too messy - Overview is done in #395
